src/Validation/tests/phpstan-bootstrap.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
```

Comment:
Would we be able to exclude these files from git archives? Would we need one of these bootstrap files for each package as well?

Reply:
The top level `.gitattributes` also needs to be updated to skip all these extra files from the full CakePHP packages.

Reply:
Should all be adjusted